tests/app/main/helpers/validation/test_dos5_declaration.py

```diff
@@ -36,16 +37,6 @@ def test_no_error_if_correct(content, submission):
     assert DOS5Validator(content, submission).errors() == {}
 
 
-def test_invalid_email_addresses_cause_errors(content, submission):
-    submission['contactEmail'] = 'not an email'
-    submission['contactEmailContractNotice'] = "not an email"
-
-    assert DOS5Validator(content, submission).errors() == {
-        'contactEmail': 'invalid_format',
-        'contactEmailContractNotice': 'invalid_format',
-    }
```

Comment:
We still have tests for this behaviour somewhere else right?

Reply:
Yes - we fall back to the superclass behaviour, which has a test: https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/tests/app/main/helpers/validation/test_dos_declaration.py#L227